### DISCUSSION TITLE
R68S Docker 怎么扩容呢？

### DISCUSSION THREAD

**Discussion by meioxyz** [coolsnowwolf/lede#9541] [R68S / 电犀牛专区]
有没有R68S Docker 扩容的详细教程呢？

**Comment by SLiang-liao** [comment]
直接将空闲的空间挂载到 /opt/docker

**Comment by daghy** [comment]
https://post.smzdm.com/p/awx44prk/参考这篇文章最后边扩容教程。基本一模一样的。

**Comment by smomop** [comment]
最新版的在磁盘管理里修改磁盘，找到13G的分区，选择格式化ext4，最后在挂载点挂载到/opt/docker,完事
